project/OSGi.scala

```diff
@@ -138,9 +138,22 @@ object OSGi {
       scalaImport(scalaVersion))
   def pekkoImport(version: String, packageName: String = "org.apache.pekko.*") = {
     val versionComponents = version.split('.')
-    val nextMinorVersion = versionComponents(1).toInt + 1
-    versionedImport(packageName, s"${versionComponents.head}.${versionComponents(1)}",
-      s"${versionComponents.head}.$nextMinorVersion")
+    if (versionComponents.length < 2) {
```

Comment:
Can we added a comment here explaining the context of what's going on (I.e. summary of PR)